tests/test_utils_persistence.py

```diff
@@ -33,6 +33,19 @@ def test_swallows_exceptions_making_new_directories(mocker):
     os.makedirs.assert_called_once()
 
 
+def test_is_yml_identifies_yml():
+    zip_dir = os.path.join('path', 'to', 'dir', 'file.yml')
```

Comment:
`zip_dir` feels like a misleading name. Although you don't really need the variable:

```python
assert persistence.is_yml(os.path.join('path', 'to', 'dir', 'file.yml'))
```

Reply:
copy pasta fail, typical >_>'